src/app/_components/attachment/attachment.component.ts

```diff
@@ -264,13 +264,14 @@ export class AttachmentComponent implements OnInit {
           
           this.att_id = fileRes.ID;
 
-          this.pConn$.attachmentsInfo = {
+          const reqObj = {
             type: "File",
             attachmentFieldName: this.att_valueRef,
             category: this.att_categoryName,
-            ID: fileRes.ID    // 8.7 - ID is at the top-level now.
+            ID: fileRes.ID
           };
-  
+          this.pConn$.attachmentsInfo = this.PCore$.getPCoreVersion()?.includes('8.8') ? [reqObj] : reqObj;
```

Comment:
Does this logic work for the next versions of infinity? I think we should modify this logic to handle two cases

versions below 8.7
versions above 8.7

Reply:
updated the PR by modifying the condition to support 8.7 and above versions, for versions below 8.7 we have different branch so no need to handle